internal/output/column.go

```diff
@@ -1,7 +1,352 @@
 package output
 
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"reflect"
+	"strings"
+	"time"
+
+	"github.com/hazelcast/hazelcast-go-client/serialization"
+	"github.com/hazelcast/hazelcast-go-client/types"
+
+	"github.com/hazelcast/hazelcast-commandline-client/errors"
+	iserialization "github.com/hazelcast/hazelcast-commandline-client/internal/serialization"
+)
+
 type Column struct {
 	Name  string
 	Type  int32
 	Value any
 }
+
+func NewStringColumn(value string) Column {
+	return Column{Type: iserialization.TypeString, Value: value}
+}
+
+func NewNilColumn() Column {
+	return Column{Type: iserialization.TypeNil}
+}
+
+func NewSkipColumn() Column {
+	return Column{Type: iserialization.TypeSkip}
+}
+
+func NewKeyColumn(kt int32, key any) Column {
+	return Column{
+		Name:  NameKey,
+		Type:  kt,
+		Value: key,
+	}
+}
+
+func NewKeyTypeColumn(kt int32) Column {
+	return Column{
+		Name:  NameKeyType,
+		Type:  iserialization.TypeString,
+		Value: iserialization.TypeToString(kt),
+	}
+}
+
+func NewValueColumn(vt int32, value any) Column {
+	return Column{
+		Name:  NameValue,
+		Type:  vt,
+		Value: value,
+	}
+}
+
+func NewValueTypeColumn(vt int32) Column {
+	return Column{
+		Name:  NameValueType,
+		Type:  iserialization.TypeString,
+		Value: iserialization.TypeToString(vt),
+	}
+}
+
+func (co Column) SingleLine() (s string) {
+	/*
+		defer func() {
+			if e := recover(); e != nil {
+				s = ValueNotDecoded
+			}
+		}()
+	*/
```

Comment:
Should we remove this?

Reply:
Uncommented at: https://github.com/hazelcast/hazelcast-commandline-client/pull/162/commits/e7f4a6f200af7ce276b0b087c2f91f4340ef8140